spec/models/snap_application_spec.rb

```diff
@@ -62,4 +62,52 @@
       end
     end
   end
+
+  describe ".enqueue_faxes" do
+    it "schedules jobs for signed, unfaxed application updated awhile ago" do
+      signed_unfaxed_updated_awhile_ago = create(:snap_application,
```

Comment:
it might be easier to read if we put each of these vars into a private method. Just to make the spec easier to read. No biggie either way

Reply:
Good point! I normally use `let!` for that, since that's basically what `let!` does.

Reply:
yes for sure! Joel and I have a weird allergic reaction to `let` that might not entirely be fair but is very real. I always go back to this cheeky blog post when discussing it https://robots.thoughtbot.com/lets-not

So for now I'd love to do private methods instead of `let`, but inline is also 💯